src/common/objects/Group.cpp

```diff
@@ -33,9 +33,18 @@ void Group::draw(sf::RenderTarget& render_target) {
     }
     setOutlineColor(outline_color);
 
+    std::vector<std::pair<sf::Vector2f, sf::Color>> direction_color_pairs;
+    direction_color_pairs.reserve(m_targetDirections.size());
+    for (auto& direction : m_targetDirections) {
+        direction_color_pairs.push_back(
+            std::make_pair(direction, RenderingDef::DIRECTION_LINE_DEFAULT_COLOR));
```

Comment:
Pass players' desired resource's color instead of `DIRECTION_LINE_DEFAULT_COLOR`
Need to create a mechanism to input and store this value